macros/contextInteger.pl

```diff
@@ -182,9 +182,11 @@ sub phi {
   my $a = abs(shift);
   $self->Error("Cannot phi on Zero.") if $a == 0;
   $result = $a; $n = $a;
-  for (my $i = 2; ($i ** 2) < $n; $i++) {
-    while ($n % $i == 0) {
-      $n /= $i;
+  for (my $i = 2; ($i ** 2) <= $n; $i++) {
+    if ($n % $i == 0) {
```

Comment:
This would be better as
```suggestion
    next unless ($n % $i == 0);
```

Reply:
Thanks for the suggestion!  Fixed.